JacquardToolkit/JacquardToolkit/JacquardService.swift

```diff
@@ -209,33 +217,38 @@ extension JacquardService: CBPeripheralDelegate {
             input_data![JSConstants.JSNumbers.ForceTouch.fullThreadCount - JSConstants.JSNumbers.ForceTouch.threadCount + i] = threadReadings[i] as NSNumber
         }
         
-        let prediction = try? forceTouchModel.prediction(input: NewGestureClassifier_RC2Input(_15ThreadConductivityReadings: input_data!))
-        print("Prediction Result: \((prediction?.output["ForceTouch"])!)")
-        
+        let prediction = try? forceTouchModel.prediction(input: ForcetouchInput(_15ThreadConductivityReadings: input_data!))
+//        print("Prediction Result: \((prediction?.output["ForceTouch"])!)")
+
         if forceTouchTurnedEnabled {
-            if ((prediction?.output["ForceTouch"])! > 0.7) {
-                forceTouchTurnedEnabled = false
-                delegate?.didDetectForceTouchGesture()
-                return true
+            if ((prediction?.output["ForceTouch"])! > forceTouchDetectionThreshold) {
```

Comment:
Let's take this "ForceTouch" string out of JacquardService.swift and put it into JSConstants.swift